src/Parser/Parse.php

```diff
@@ -195,14 +194,17 @@ public function parse(): bool
                             if (is_array($quill['insert']) === true) {
                                 if (array_key_exists('image', $quill['insert']) === true) {
                                     $this->image($quill);
-                                } else if (array_key_exists('video', $quill['insert']) === true) {
-                                    $this->video($quill);
+                                } else {
+                                    if (array_key_exists('video', $quill['insert']) === true) {
```

Comment:
<sub>That's a weird javascript notation :/</sub>

Reply:
See previous answer.